.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Continuous Integration
```

Comment:
we need to get consistent in our GHA names... lets pick this up in next huddle to conclude on naming for GHA's